impl/src/planning/subplanners.rs

```diff
@@ -122,6 +122,8 @@ impl<'planner> WorkspaceSubplanner<'planner> {
       GenMode::Vendored => {
         checks::check_all_vendored(self.crate_catalog.entries(), &self.settings.workspace_path)?;
       },
+      // TODO: Remove when `Unspecified` is removed.
```

Comment:
Do you know if serde support requiring a field? I looked earlier and didn't see anything... though this might be a dumb question that's obvious if you've used it recently.

If they don't support required fields, then we may never be able to remove this.

Reply:
Good question, I could have sworn it did but now I'm not so sure... Need to look

Reply:
Yeah, good call, I don't think it does support flagging something as a required field. I've updated the code to account for this.